src/main/groovy/com/gorylenko/writer/SkipPropertiesCommentsOutputStream.groovy

```diff
@@ -1,13 +1,16 @@
 package com.gorylenko.writer
 
+import groovy.transform.CompileStatic
+
 /**
  * Remove all lines start with # and end with LF (\n)
  */
+@CompileStatic
 class SkipPropertiesCommentsOutputStream extends FilterOutputStream {
 
-    private static int START_COMMENT_CHAR1 = '#'
-    private static int START_COMMENT_CHAR2 = '!'
-    private static int LINE_END_CHAR = '\n'
+    private static int START_COMMENT_CHAR1 = '#' as char
+    private static int START_COMMENT_CHAR2 = '!' as char
+    private static int LINE_END_CHAR = '\n' as char
 
```

Comment:
Why not define `char` type directly, it can be interacted with `int`.

Reply:
I changed them to `char` and made them `final`